roboteam_ai/src/control/ControlUtils.cpp

```diff
@@ -10,6 +10,14 @@ namespace rtt {
 namespace ai {
 namespace control {
 
+std::map<int, double> ControlUtils::genevaMap = {
+        {1, -20.0 / 180 * M_PI},
```

Comment:
A comment stating that these are geneva states mapped to their angles in radians would be nice

Reply:
DOne